1e/manifest.json

```diff
@@ -0,0 +1,46 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "d86565ae-ca60-45c2-a0b9-944ec1e05a15",
+  "app_id": "one-e",
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Send your 1E product metrics to Datadog.",
+    "title": "1E Integration",
+    "media": [],
+    "classifier_tags": [
+      "Supported OS::Linux",
+      "Supported OS::Windows",
+      "Supported OS::macOS",
+      "Offering::Integration",
+      "Category::Metrics"
+    ]
+  },
+  "assets": {
+    "integration": {
+      "source_type_name": "1e",
```

Comment:
```suggestion
      "source_type_name": "one-e",
```

Reply:
That change was done in last PR as well but was reverted back due to failing of some validation checks

![image](https://user-images.githubusercontent.com/115262928/209083890-95f704ed-b1ae-463e-be84-564989e3c546.png)

Reply:
Yes this needs to be changed

Reply:
@Sid-1E Can you please change this to be one-e?

Reply:
@emarsha94 changed to one-e